pkg/operator/operator.go

```diff
@@ -94,12 +93,6 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
-	rookAgent := agent.New(o.context.Clientset)
```

Comment:
The rook agent is needed to help orchestrate the kubernetes volume plugins with mount, attach, etc. Why remove it?

Reply:
When I grepped the code I couldn't see this instance in the operator being used anywhere?  (the instantiation in cmd/rook/agent.go obviously makes sense, this one seemed dead).

I'm a newbie to this codebase though so I have a feeling you're about to explain to me what I've missed :-)

Reply:
This definitely looks in use to me still @jcsp, please do not remove this.

This here calls this `Run()` function: https://github.com/rook/rook/blob/master/cmd/rook/operator.go#L79

Reply:
Ah, I get it now -- this class isn't the agent, it's a component of the operator that goes and runs the agent elsewhere.